cognee/tests/integration/run_toy_tasks/run_tasks_simple_test.py

```
@@ -0,0 +1,41 @@
+import asyncio
+
+from cognee.modules.pipelines.tasks import Task, TaskConfig
+from cognee.modules.pipelines.operations.run_tasks_v2 import run_tasks_base
+
+
+async def run_and_check_tasks():
+    def number_generator(num, context={}):
+        for i in range(num):
+            yield i + 1
+
+    async def add_one(num, context={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)